tox.ini

```diff
@@ -87,19 +89,23 @@ deps =
     -r{toxinidir}/server/requirements.txt
     -r{toxinidir}/server/test-requirements.txt
 commands =
-    pytest ./lib/pbench/test/unit/common
-    pytest ./lib/pbench/test/unit/server 
-    pytest ./lib/pbench/test/functional
+    pytest --cov --cov-append \
+      --cov-report html:{envdir}/cov/html \
```

Comment:
So the server report goes to the default `{envdir}` while the agent report goes to `/tmp/${USER}}`?? I suppose they can't be placed side-by-side because we apparently don't control the name, but the asymmetry still seems a bit odd. E.g., why not `/tmp/${USER}/server` and `/tmp/${USER}/agent` or something? But ... not a big deal, just my confusion.

Reply:
Bug.